packages/xdl/src/Exp.ts

```diff
@@ -184,16 +186,16 @@ async function initGitRepoAsync(root: string) {
 async function installDependenciesAsync(projectRoot: string, packageManager: 'yarn' | 'npm') {
   Logger.global.info('Installing dependencies...');
 
+  const options = { cwd: projectRoot };
   if (packageManager === 'yarn') {
-    await spawnAsync('yarnpkg', ['install'], {
-      cwd: projectRoot,
-      stdio: 'inherit',
-    });
+    const yarn = new YarnPackageManager(options);
+    const version = await yarn.versionAsync();
+    if (semver.satisfies(version, '>=2.0.0-rc.24')) {
+      await fs.writeFile(path.join(projectRoot, '.yarnrc.yml'), 'nodeLinker: node-modules\n');
```

Comment:
we should probably:
- let the user know that we're doing this
- bail out if the file already exists, check if it contains `nodeLinker: node-modules`, warn them if not